packages/xrpl/src/models/transactions/XChainAddAccountCreateAttestation.ts

```diff
@@ -0,0 +1,225 @@
+import { ValidationError } from '../../errors'
+import { Amount, XChainBridge } from '../common'
+
+import {
+  BaseTransaction,
+  isAmount,
+  isXChainBridge,
+  validateBaseTransaction,
+} from './common'
+
+/**
+ * The XChainAddAccountCreateAttestation transaction provides an attestation
+ * from a witness server that a {@link XChainAccountCreateCommit} transaction
+ * occurred on the other chain.
+ *
+ * @category Transaction Models
+ */
+export interface XChainAddAccountCreateAttestation extends BaseTransaction {
+  TransactionType: 'XChainAddAccountCreateAttestation'
+
+  /**
+   * The amount committed by the {@link XChainAccountCreateCommit} transaction
+   * on the source chain.
+   */
+  Amount: Amount
+
+  /**
+   * The account that should receive this signer's share of the SignatureReward.
+   */
+  AttestationRewardAccount: string
+
+  /**
+   * The account on the door account's signer list that is signing the transaction.
+   */
+  AttestationSignerAccount: string
+
+  /**
+   * The destination account for the funds on the destination chain.
+   */
+  Destination: string
+
+  /**
+   * The account on the source chain that submitted the {@link XChainAccountCreateCommit}
+   * transaction that triggered the event associated with the attestation.
+   */
+  OtherChainSource: string
+
+  /**
+   * The public key used to verify the signature.
+   */
+  PublicKey: string
+
+  /**
+   * The signature attesting to the event on the other chain.
+   */
+  Signature: string
+
+  /**
+   * The signature reward paid in the {@link XChainAccountCreateCommit} transaction.
+   */
+  SignatureReward: Amount
+
+  /**
+   * A boolean representing the chain where the event occurred.
```

Comment:
Is it worth saying which one is true / false? (I can't off the top of my head remember if 0 or 1 is true)

Reply:
`0` is always false and `1` is always true. I don't think it's necessary to specify given that it's standard.